homeassistant/components/vallox/sensor.py

```diff
@@ -157,7 +156,7 @@ class ValloxSensorEntityDescription(SensorEntityDescription):
         metric_key="A_CYC_EXTR_FAN_SPEED",
         icon="mdi:fan",
         state_class=SensorStateClass.MEASUREMENT,
-        native_unit_of_measurement=FREQUENCY_HERTZ,
+        native_unit_of_measurement="RPM",
```

Comment:
The code changes look good, but looking at the code base, I wonder if a global constant shouldn't be created for this in `homeassistant.components.const`. It would help standardise in the code base.

Then as a follow-up PR the following components could be adjusted to use the new constant: `baf`, `comfoconnect`, `danfoss_air`, `glances`, `isy994`, `qnap_qsz`, `smarty`, `system_bridge`, and `xiaomi_miio`.
I even wonder if a new device class might be useful for this (sample in #77666)...

Reply:
A global constant could indeed be added, but that's a separate PR (that could be done either before or after this one), isn't it?